src/fides/api/task/filter_results.py

```diff
@@ -37,6 +38,11 @@ def filter_data_categories(
         if not results:
             continue
 
+        # Skip manual task data - it doesn't need filtering since it's controlled by field definitions
+        if f":{ManualTaskAddress.MANUAL_DATA_COLLECTION}" in node_address:
+            filtered_access_results[node_address].extend(results)
+            continue
```

Comment:
Skipping data category filtering for now. Any manual inputs provided to a privacy request will be included in the DSR package.